src/Hl7.Fhir.Base/ElementModel/ScopedNodeExtensions.cs

```diff
@@ -136,7 +136,7 @@ public static ResourceIdentity MakeAbsolute(this ScopedNode node, ResourceIdenti
             string? url = element switch
             {
                 { Value: string s } => s,
-                { InstanceType: FhirTypeConstants.REFERENCE } => element.ParseResourceReference()?.Reference,
+                { InstanceType: FhirTypeConstants.REFERENCE } => element.ParseResourceReference<ITypedElement>()?.Reference,
```

Comment:
Was this necessary? It couldn't be derived from the type of `element` here? If so, then this is a breaking change to the public interface, right?